### ISSUE TITLE
Playwright: create test to iterate through all sidebar menu and submenu items

### ISSUE THREAD

**Issue by worldomonation** [Automattic/wp-calypso#55695]
### Details

This idea arose in p1629759153112200-slack-C01QG4Y91RR, after #54987 was merged into trunk.

Such a test could serve multiple purposes:

1. check that all pages accessible from the sidebar are loading and indeed accessible.
2. serve as verification of any future `SidebarComponent` changes.

If the purpose was for 2. alone, a better solution is to implement a Jest unit test for the `calypso-e2e` package. However, point 1. is a worthwhile task to validate on the e2e side of things.

### Checklist

- [ ] implement this as a Jest unit test or part of the e2e spec?
- [ ] how to ensure all items are navigated without using a list of menu/submenus?

**Comment by worldomonation** [comment]
I still think there is value to this.

**Comment by worldomonation** [comment]
Doubtful if this is useful, so assigning Low priority.

**Comment by worldomonation** [comment]
I don't think this is useful anymore, so closing.
